pallets/currency/Cargo.toml

```diff
@@ -7,29 +7,29 @@ name = "currency"
 version = "1.0.13"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive", "max-encoded-len"] }
-scale-info = { version = "2.0.0", default-features = false, features = ["derive"] }
+codec = { workspace = true, features = ["derive", "max-encoded-len"] }
+scale-info = { workspace = true, features = ["derive"] }
 
-sp-std = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
-frame-support = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
-frame-system = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
+sp-std = { workspace = true }
+frame-support = { workspace = true }
+frame-system = { workspace = true }
 
 # Parachain dependencies
 primitives = { package = "spacewalk-primitives", path = "../../primitives", default-features = false }
 
 # Orml dependencies
-orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v1.1.0", default-features = false }
-orml-currencies = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v1.1.0", default-features = false }
-orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v1.1.0", default-features = false }
+orml-tokens = { workspace = true, default-features = true }
```

Comment:
You are assigning `default-features = true` everywhere, is this intentional or do you want to do `default-features = false`?

Reply:
Intentional, assigning `default_features = false` on the module but not on the workspace is overridden.

Reply:
But in this case here, you are using the `default-features = true` on a dependency and not a dev-dependency which seems wrong to me, considering that it will be included in the wasm runtime like this?

Reply:
Oh!! In this case is absolutely wrong, of course. The general case holds but this is doing the opposite.